app/models/vm_or_template/operations/relocation.rb

```diff
@@ -1,4 +1,16 @@
 module VmOrTemplate::Operations::Relocation
+  extend ActiveSupport::Concern
+
+  included do
+    supports_not :live_migrate
+    supports_not :warm_migrate
+    supports :migrate do
+      if blank? || orphaned? || archived?
+        unsupported_reason_add(:migrate, N_("Migrate operation in not supported."))
```

Comment:
Same here...I thikn this can be `_()`

Reply:
I am also questioning having a reason that does not explain WHY something is not supported

Reply:
I agree

Reply:
addressed